app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.java

```diff
@@ -0,0 +1,100 @@
+package org.oppia.app.databinding;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.PluralsRes;
+import androidx.databinding.BindingAdapter;
+import org.oppia.app.R;
+import org.oppia.util.system.OppiaDateTimeFormatter;
+import java.util.Locale;
+import java.util.concurrent.TimeUnit;
+
+/** Holds all custom binding adapters that bind to [TextView]. */
+public final class TextViewBindingAdapters {
+
+  private static int MINUTE_MILLIS = (int) TimeUnit.MINUTES.toMillis(1);
+  private static int HOUR_MILLIS = (int) TimeUnit.HOURS.toMillis(1);
+  private static int DAY_MILLIS = (int) TimeUnit.DAYS.toMillis(1);
+
+  /** Binds date text with relative time. */
+  @BindingAdapter("profile:created")
+  public static void setProfileDataText(@NonNull TextView textView, long timestamp) {
+    OppiaDateTimeFormatter oppiaDateTimeFormatter = new OppiaDateTimeFormatter();
+    String time = oppiaDateTimeFormatter.formatDateFromDateString(
+        OppiaDateTimeFormatter.DD_MMM_YYYY,
+        timestamp,
+        Locale.getDefault()
+    );
+    textView.setText(textView.getContext().getString(
+        R.string.profile_edit_created,
+        time
+    ));
+  }
+
+  @BindingAdapter("profile:lastVisited")
+  public static void setProfileLastVisitedText(@NonNull TextView textView, long timestamp) {
+    textView.setText(
+        String.format(
+            textView.getContext().getString(R.string.profile_last_used) + " " + getTimeAgo(
```

Comment:
Unrelated to your PR, but we should never use concatenation for strings. Please file a bug (outside of the Bazel project) & add a TODO here to change this to be a proper multi-variable string so that it can be properly RTL translated.

Feel free to resolve this comment thread once the issue & TODO comment are added.

Reply:
Done!